.github/workflows/ruby.yml

```diff
@@ -6,17 +6,33 @@
 # For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
 
 name: Ruby
-on: [push, pull_request]
+
+on: [push]
 jobs:
+  check_react_and_ujs:
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby: [2.7]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler: 2.4.9
+          ruby-version: ${{ matrix.ruby }}
+      - uses: actions/setup-node@v3
+      - run: bundle install
+      - run: yarn
+      - run: bundle exec rake react:update
+      - run: bundle exec rake ujs:update
+      - run: ./check_for_uncommitted_files.sh
```

Comment:
I think it is better to extract this logic into another rake task and use it in both:
- Release script
- CI

Is will be DRY and also prevents adding bash script in the root of the project.

Reply:
I'm not convinced.